script/Ciel/Day 9/09_01_CIEL09_8.txt

```diff
@@ -273,12 +273,13 @@ Senpai says, with her face facing downward.
 [sha:07446ff9b5664484dd310b818c7b78ef9a831fe0]{
 -- Page 714, Offset 23992.
 -- なんとなしに言葉がない。
-I don't know why but I find myself lost for words.
+I don't know why, but I'm at a loss for words.
 }
 [sha:1ce0c3c9bd163b8f89146853f60255434d76ae52]{
 -- Page 714, Offset 23993.
 -- 気がつけば体が触れ合うほど近くて、先輩の体温がほのかに感じられて、ムズムズする。
-Without even realizing it, our bodies grow enough to be touching, and Senpai's faint warmth makes me all jittery.
+// sex
```

Comment:
is this what sex is? i tried to google it but i couldn't find any info

Reply:
this is what my teachers taught me